selenium_test.py

```diff
@@ -0,0 +1,37 @@
+from datetime import time
+import pytest
+import app
+import time
+from selenium import webdriver
+from threading import Thread
+
+@pytest.fixture(scope='module')
+def test_app():
+    # construct the new application
```

Comment:
This test fails for me because it doesn't load any environment variables before running the app - this should probably load some in similarly to the integration test, and there should be some instructions on how to setup for this (e.g. adding test credentials into the .env.test file) and run it in your Readme

Reply:
This has been addressed in this commit - f4ce91a